public/views/editProfile.php

```diff
@@ -1,12 +1,66 @@
 <?php
-$SessionController = new SessionController;
+$SessionController = new SessionController();
+$userIsAuthenticated = $SessionController::isLogged();
+$userInfo = $SessionController->unserializeUser();
 
 if ($SessionController::isLogged() === false) {
     $SessionController->redirectToHome();
 }
 
+require_once "public/views/components/card.php";
+require_once "public/views/components/buttonRedirect.php";
+require_once "public/views/components/avatarProfile.php";
+require_once "public/views/components/button.php";
+require_once "public/views/components/form.php";
+require_once "public/views/components/textarea.php";
+
 $userInfo = $SessionController->unserializeUser();
 
+// $description = $user->getDescription() ?: 'Ten użytkownik nie dodał jeszcze swojego opisu...';
```

Comment:
$user nie istnieje, chcemy tutaj zaczepić o naszą zmienną $userInfo

Wyglądało by to tak:
`$description = $userInfo->getDescription() ?: '';`

Tekst 'Ten użytkownik...' nie ma sensu, gdyż przy każdej edycji musiałby być kasowany gdyby ktoś po prostu opisu nie chciał.